server/handlers_v2.go

```diff
@@ -438,6 +544,71 @@ func (server HTTPServer) getImpactingRecommendations(
 	return aggregatorResponse.Recommendations, nil
 }
 
+// getClustersAndRecommendations retrieves a list of recommendations from aggregator based on the list of clusters
+func (server HTTPServer) getClustersAndRecommendations(
+	writer http.ResponseWriter,
+	orgID ctypes.OrgID,
+	userID ctypes.UserID,
+	clusterList []ctypes.ClusterName,
+) (ctypes.ClusterRecommendationMap, error) {
+
+	var aggregatorResponse struct {
+		Clusters ctypes.ClusterRecommendationMap `json:"clusters"`
+		Status   string                          `json:"status"`
+	}
+
+	aggregatorURL := httputils.MakeURLToEndpoint(
+		server.ServicesConfig.AggregatorBaseEndpoint,
+		ira_server.ClustersRecommendationsListEndpoint,
+		orgID,
+		userID,
+	)
+
+	jsonMarshalled, err := json.Marshal(clusterList)
+	if err != nil {
+		log.Error().Err(err).Msgf("problem unmarshalling cluster list")
+		handleServerError(writer, err)
+		return nil, err
+	}
+
+	// #nosec G107
+	aggregatorResp, err := http.Post(aggregatorURL, JSONContentType, bytes.NewBuffer(jsonMarshalled))
+	if err != nil {
+		log.Error().Err(err).Msgf("problem getting response from aggregator")
```

Comment:
another one :)
(btw it might be caught by goconst checker)

Reply:
thanks, gotta start paying attention to these ones